buildroot/share/PlatformIO/variants/MARLIN_G0B1RE/variant_MARLIN_STM32G0B1RE.cpp

```diff
@@ -130,14 +130,7 @@ WEAK void SystemClock_Config(void)
   RCC_PeriphCLKInitTypeDef PeriphClkInit = {0};
 
   // Reset clock registers (in case bootloader has changed them)
-  RCC->CR |= RCC_CR_HSION;
-  while (!(RCC->CR & RCC_CR_HSIRDY))
-      ;
-  RCC->CFGR = 0x00000000;
-  RCC->CR = RCC_CR_HSION;
-  while (RCC->CR & RCC_CR_PLLRDY)
-      ;
-  RCC->PLLCFGR = 0x00001000;
+  SystemInit();
```

Comment:
@thisiskeithb I recently updated my SKR Mini E3 v3.0 to the lastest bugfix 2.1 and this change made the printer irresponsive, I just get a blank screen after boot

I have no idea what this code does, but after reverting to the previous version (keeping all other changed), it booted again.

Reply:
> @thisiskeithb I recently updated my SKR Mini E3 v3.0 to the lastest bugfix 2.1 and this change made the printer irresponsive, I just get a blank screen after boot
> 
> I have no idea what this code does, but after reverting to the previous version (keeping all other changed), it booted again.

@mistic100: Please create a proper [Bug Report](https://github.com/MarlinFirmware/Marlin/issues/new/choose) and be sure to attach your configs.

I tested this PR on an SKR Mini E3 V3 and did not have any issues.